src/squid.md

```diff
@@ -0,0 +1,243 @@
+
```

Comment:
> Francesco: gfm supports - in fact, requires - metadata in the form of [Front Matter](https://jekyllrb.com/docs/front-matter/)

Thank you for that research. Hm... AFAICT, the current contents of this file in this PR does not contain metadata in the form of Front Matter. On the other hand, this file is processed using a "gfm" filter. Is this file missing a "required"  metadata section? Does Pandoc supply its own metadata in such cases (and, if "yes", is that supplied-by-default metadata correct for our use cases?).

Reply:
Let me clarify:
- jeckyll / gfm / github pages requires front matter (this is why all pages in our wiki have it), in some cases there's a plugin specifying defaults which makes it moot
- gfm may allow front matter, but how it is interpreted seems to depend on the implementation

I suppose the only way to know is by trying

Reply:
> `pandoc` documentation indicates that it supports Front Matter. However when I tried to use it for this manual document it did not work. Only supplying the values via pandoc command line arguments produced the required man syntax correctly.

Was that failed attempt specific to `gfm`? Or did it fail with some other filter formats you have tried during this PR lifetime? I am trying to understand if this lack of Front Matter support (in pandoc context) is specific to the `gfm` filter...

Reply:
You are right here. It does seem to be fine with specifically the GFM format, but not with others.
Updated in 5f63c13e56